api/envoy/config/bootstrap/v4alpha/bootstrap.proto

```diff
@@ -137,7 +138,10 @@ message Bootstrap {
   Admin admin = 12;
 
   // Optional overload manager configuration.
-  overload.v3.OverloadManager overload_manager = 15;
+  overload.v3.OverloadManager overload_manager = 15 [
```

Comment:
The plan is to fill out these annotations and the manifest examples for the remaining default edge examples if we agree on this approach.